openshift/dynamic/client.py

```diff
@@ -174,7 +179,7 @@ def replace(self, resource, body=None, name=None, namespace=None, **kwargs):
 
     @meta_request
     def patch(self, resource, body=None, name=None, namespace=None, **kwargs):
-        body = body or {}
+        body = self.serialize_body(body)
```

Comment:
I'm not sure I understand the differences between how body is set for replace vs patch. What's up with that?

Reply:
Cool